internal/runbits/auth/login.go

```diff
@@ -222,9 +223,14 @@ func authenticateWithBrowser(out output.Outputer, auth *authentication.Auth, pro
 			return errs.Wrap(err, "Verification URL is not valid")
 		}
 
-		signupURL, err := url.Parse(constants.PlatformSignupURL)
+		apiHost := constants.DefaultAPIHost
+		if hostOverride := os.Getenv(constants.APIHostEnvVarName); hostOverride != "" {
+			apiHost = hostOverride
+		}
+		fullURL := "https://" + apiHost + constants.PlatformSignupPath
+		signupURL, err := url.Parse(fullURL)
```

Comment:
I think this is a good opportunity to add a function to the `api` package like `GetPlatformURL`. That way we can try to centralize usage of the host override env var to one package, rather than potentially sprinkle it throughout the codebase.